src/index.js

```diff
@@ -1 +1,94 @@
 import './style.css';
+import display from './modules/display.js';
+import logoImage from './assets/Logo.png';
+import { updateLikes, postLikes } from './modules/displaylikes.js';
```

Comment:
- [ ] The likes functionality has some bugs.  Let me explain with a video:

https://share.getcloudapp.com/Z4u1EJ9y

Therefore, please correct this functionality, so the likes buttons work as intended.

That way, your project will be rock solid! 💪

Reply:
Resolved ✔️